src/twisted/internet/test/test_base.py

```diff
@@ -369,3 +369,65 @@ def test_str(self):
             "traceback at creation:".format(id(dc)))
         self.assertRegex(
             str(dc), expectedRegexp)
+
+
+
+class TestSpySignalCapturingReactor(ReactorBase):
+
+    """
+    Subclass of ReactorBase to capture signals delivered to the
+    reactor for inspection.
+    """
+
+    def installWaker(self):
+        """
+        Required method, unused.
+        """
+        pass
```

Comment:
I was thinking about trying to add the ability to check a patchfile (similar to the Linux kernel's checkpatch utility) to twistedchecker.  The benefit there being that it would potentially ignore previous errors in other unrelated parts of the file.  If I go down that road I'll look at adding this too if nobody beats me to it. 

Will correct in V3.